packages/extension-ui/src/Popup/About.tsx

```diff
@@ -32,110 +20,110 @@ function About({ className }: Props): React.ReactElement<Props> {
+              link={LINKS.TERMS_OF_SERVICE}
+              rightIcon='link'
+              title={t<string>('Terms of Service')}
+            />
+            <LinksList.Item
+              link={LINKS.PRIVACY_POLICY}
+              rightIcon='link'
+              title={t<string>('Privacy Policy')}
+            />
+          </LinksList.Group>
+          <LinksList.Group>
+            <LinksList.Item
+              link={LINKS.MAIN_WEBSITE}
+              rightIcon='link'
+              title={t<string>('Visit Website')}
+            />
+          </LinksList.Group>
+        </Links>
+      </Container>
     </>
   );
-}
+};
 
-export default React.memo(
-  styled(About)(
-    ({ theme }: Props) => `
-  color: ${theme.textColor};
+export default About;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  color: ${({ theme }) => theme.textColor};
   height: 100%;
-  height: calc(100vh - 2px);
   overflow-y: scroll;
   scrollbar-width: none;
-  margin-top: 38px;
-      
+
   &::-webkit-scrollbar {
     display: none;
   }
+`;
 
-  .icon {
-    width: 20px;
-    height: 20px;
-    background: ${theme.primaryColor};
-  }
+const Hero = styled.div``;
 
-  .aleph-mark {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    margin: 0 auto;
-  }
+const AlephLogo = styled.img`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0 auto;
+`;
 
-  .text {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-    margin-top: 34px;
-    margin-bottom: 36px;
+const HeroTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 34px;
+  margin-bottom: 36px;
+`;
 
-    .heading {
-      font-family: ${theme.secondaryFontFamily};
-      font-style: normal;
-      font-weight: bold;
-      font-size: 24px;
-      line-height: 118%;
-      display: flex;
-      align-items: center;
-      text-align: center;
-      letter-spacing: 0.03em;
-    }
+const Heading = styled.span`
+  font-family: ${({ theme }) => theme.secondaryFontFamily};
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 118%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  `;
 
-    .version-subtitle {
-      font-style: normal;
-      font-weight: 300;
-      font-size: 14px;
-      line-height: 145%;
-      display: flex;
-      align-items: center;
-      text-align: center;
-      letter-spacing: 0.07em;
-      color: ${theme.subTextColor}
-    }
-  }
+const Version = styled.span`
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 145%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.07em;
+  color: ${({ theme }) => theme.subTextColor}
+`;
 
-  `
-  )
-);
+const Links = styled.div``;
```

Comment:
perhaps just `div`?

Reply:
I wondered about it. I thought the way it is now is more descriptive when you read the render function. Wdyt?

Reply:
But I'm not attached to any solution at all.

Reply:
I don't mind current solution - nav would be more semantic, I think